core/src/main/scala/org/apache/spark/scheduler/EventLoggingListener.scala

```diff
@@ -90,8 +90,13 @@ private[spark] class EventLoggingListener(
    * Creates the log file in the configured log directory.
    */
   def start() {
-    if (!fileSystem.getFileStatus(new Path(logBaseDir)).isDirectory) {
```

Comment:
Note this code was broken...it would correctly fail if the log dir was a file, but would throw an FNFE if the dir didn't exist. The patch here catches the FNFE and prints the message which appears to be intended, reporting the dest is not a file in the other case.